presto-main/src/main/java/com/facebook/presto/cost/SimpleStatsRule.java

```diff
@@ -36,8 +36,22 @@ protected SimpleStatsRule(StatsNormalizer normalizer)
     @Override
     public final Optional<PlanNodeStatsEstimate> calculate(T node, StatsProvider sourceStats, Lookup lookup, Session session, TypeProvider types)
     {
-        return doCalculate(node, sourceStats, lookup, session, types)
+        Optional<PlanNodeStatsEstimate> planNodeStatsEstimate = doCalculate(node, sourceStats, lookup, session, types)
                 .map(estimate -> normalizer.normalize(estimate, node.getOutputVariables()));
+        if (node.getSources().isEmpty()) {
+            // dont do the confident check for tablescan stats
```

Comment:
If no variable statistics, we could set the confidence to be false but not sure that would be effective